MeshLib/MeshEditing/RasterDataToMesh.cpp

```diff
@@ -0,0 +1,98 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "RasterDataToMesh.h"
+
+#include "MeshLib/Node.h"
+#include "MeshLib/Elements/Element.h"
+
+namespace MeshLib
+{
+
+namespace RasterDataToMesh
+{
+
+bool checkMesh2D(MeshLib::Mesh const& mesh)
+{
+    if (mesh.getDimension() > 2)
+    {
+        ERR("This functionality is currently only available for 2D meshes.");
+        return false;
+    }
+    return true;
+}
+std::string getValidName(std::vector<std::string> const& vec_names,
+                     std::string const& array_name)
+{
+    std::string new_name = array_name;
+    std::size_t count = 1;
+    bool name_exists = std::find(vec_names.cbegin(), vec_names.cend(),
+                                 array_name) != vec_names.end();
+    while (name_exists)
+    {
+        count++;
+        new_name = array_name + std::to_string(count);
+        name_exists = std::find(vec_names.cbegin(), vec_names.cend(),
+                                array_name) != vec_names.end();
+    }
```

Comment:
I think this would be equivalent:
```suggestion
    while (std::find(vec_names.cbegin(), vec_names.cend(), array_name)
           != vec_names.end())
    {
        count++;
        new_name = array_name + std::to_string(count);
    }
```

Reply:
✔️